docs/cookbook/dynamic-configuration/03_Advanced_Datasources/01_config.yml

```diff
@@ -0,0 +1,2 @@
+template_paths:
+  - "file:///Users/mdurham/source/agents/agent_clean/agent/docs/cookbook/dynamic-configuration/03_Advanced_Datasources/01_assets"
```

Comment:
Should this have mdurham in it? :)

Reply:
Yeah, that one I couldnt do in docker so I cheated and it slipped through. Thanks for seeing